db/migrate/20170313203052_create_asset_details.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAssetDetails < ActiveRecord::Migration[5.0]
+  def change
+    create_table :asset_details do |t|
+      t.text  :description
+      t.text  :location
+      t.text  :room
+      t.text  :rack_name
+      t.text  :lowest_rack_unit
```

Comment:
not sure this is needed, but you might want to add `t.timestamps` to this table to track created and last updated dates.